docker/Dockerfile.ci

```diff
@@ -110,7 +110,9 @@ RUN curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest
 
 # Add code stub last
 COPY . /root/code/garage
-ENV PYTHONPATH=$PYTHONPATH:/root/code/garage
+RUN ["/bin/bash", "-c", "source activate garage"]
```

Comment:
These all have to be in the same RUN layer. The conda environment only lasts for a single bash session (these lines create a new session every time).